src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -536,7 +536,22 @@ public void run() {
+        GT_Values.RA.addMixerRecipe(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Bauxite, 64L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SodiumHydroxide, 16L),  GT_Utility.getIntegratedCircuit(4),  GT_Values.NI, GT_Values.NF, GT_Values.NF, GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumhydroxide, 64L), 1200, 30);
+
+        GT_Values.RA.addBlastRecipe(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumhydroxide, 6L), GT_Utility.getIntegratedCircuit(1), GT_Values.NF, GT_Values.NF, GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 6L), GT_Values.NI, 600, 120, 1300);
+        GT_Values.RA.addBlastRecipe(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SodiumAluminate, 24L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SodiumHydroxide, 3L), GT_Values.NF, GT_Values.NF, GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 24L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SodiumCarbonate, 2L), 2400, 120, 1200);
+
+        GT_Values.RA.addBlastRecipe(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 2L),  GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Carbon, 1L), GT_Values.NF, GT_Values.NF, GT_OreDictUnificator.get(OrePrefixes.ingot, Materials.Aluminium, 3L), GT_Values.NI, 900, 120, 2054);
+        GT_Values.RA.addBlastRecipe(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 6L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Cryolite, 3L), GT_Values.NF, GT_Values.NF, GT_OreDictUnificator.get(OrePrefixes.ingot, Materials.Aluminium, 6L), GT_Values.NI, 2400, 120, 963);
+
+        GT_Values.RA.addChemicalRecipeForBasicMachineOnly(GT_OreDictUnificator.get(OrePrefixes.cell, Materials.HydrofluoricAcid, 6L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumhydroxide, 1L), Materials.SaltWater.getFluid(3000L), Materials.Water.getFluid(1000L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Cryolite, 1L), ItemList.Cell_Empty.get(6), 300, 30);
+        GT_Values.RA.addChemicalRecipeForBasicMachineOnly(GT_OreDictUnificator.get(OrePrefixes.cell, Materials.SaltWater, 3L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumhydroxide, 1L), Materials.HydrofluoricAcid.getFluid(6000L), Materials.Water.getFluid(1000L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Cryolite, 1L), ItemList.Cell_Empty.get(3), 300, 30);
+        GT_Values.RA.addMultiblockChemicalRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumhydroxide, 16L), GT_Utility.getIntegratedCircuit(24)}, new FluidStack[]{Materials.HydrofluoricAcid.getFluid(96000L), Materials.SaltWater.getFluid(48000L)}, new FluidStack[]{Materials.Water.getFluid(16000L), Materials.Chlorine.getGas(16000L), Materials.Hydrogen.getGas(112000L)}, new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Cryolite, 16L)}, 600, 480);
```

Comment:
I can't seem to wrap my head around this LCR recipe. Where is that crap load of hydrogen coming from?

Reply:
Need to check why I add it. It’s just a draft so discussion need to be start soon

Reply:
indee this was wrong. And fixed